src/controllers/controllerengine.cpp

```diff
@@ -842,12 +841,6 @@ QScriptValue ControllerEngine::connectControl(
 
     if (function.isFunction()) {
         qDebug() << "Connection:" << group << name;
-        connect(cos, SIGNAL(valueChanged(double)),
-                this, SLOT(slotValueChanged(double)),
-                Qt::QueuedConnection);
-        connect(cos, SIGNAL(valueChangedByThis(double)),
```

Comment:
Some scripts expect ```set()``` will call their handler -- I think. Switching to trigger only may not be enough.